app/app/Board.js

```diff
@@ -187,4 +187,60 @@ export class Board {
 
         this.deleteControls(controlsToDelete);
     }
-}
+
+    /**
+     * Initiate Selection Area drawing process.
+     * @param x
+     * @param y
+     */
+    startDrawingSelectionArea(x, y) {
+        this.isDrawingSelectionArea = true;
+
+        this.startXSelectionArea = x;
+        this.startYSelectionArea = y;
+
+        this.selectionArea = this.paper.rect(x, y, 0, 0);
+        this.selectionArea.remove();
```

Comment:
Thanks for the hint.

I've committed fix for this.

But, could you explain, why this is so?